GNUmakefile

```diff
@@ -138,7 +138,11 @@ update-tf-compatibility-matrix: ## Update Terraform Compatibility Matrix documen
 .PHONY: update-changelog-unreleased-section
 update-changelog-unreleased-section:
 	./scripts/update-changelog-unreleased-section.sh
-
+  
 .PHONY: generate-changelog-entry
 generate-changelog-entry:
 	./scripts/generate-changelog-entry.sh
+
+.PHONY: check-changelog
```

Comment:
nit: would `check-changelog-entry-files` be more explicit?

Reply:
sounds good, i'll change to check-changelog-entry-file in singular

Reply:
done